src/displayapp/screens/Timer.cpp

```diff
@@ -74,6 +74,15 @@ Timer::Timer(DisplayApp* app, Controllers::TimerController& timerController)
     createButtons();
   }
 
+  btnRepeat = lv_btn_create(lv_scr_act(), nullptr);
+  btnRepeat->user_data = this;
+  lv_obj_set_event_cb(btnRepeat, btnEventHandler);
+  lv_obj_align(btnRepeat, lv_scr_act(), LV_ALIGN_IN_BOTTOM_RIGHT, 0, -10);
+  lv_obj_set_height(btnRepeat, 40);
+  txtRepeat = lv_label_create(btnRepeat, nullptr);
+  lv_label_set_text(txtRepeat, "Repeat");
+  // Always starts off not repeating so color is already what it should be.
+
```

Comment:
It would make sense to make this button toggleable/checkable in LVGL with `lv_btn_set_checkable(btnRepeat, true);`. You can take a look at QuickSettings for reference. Then you won't need the `repeating` variable.

https://github.com/InfiniTimeOrg/InfiniTime/blob/26ae828e393d28390c5ea718bec9d4c1a96435ac/src/displayapp/screens/settings/QuickSettings.cpp#L82